### PR TITLE
Added notifications on creating collections and adding photos to them

### PR DESCRIPTION
**Pull request by aOrro**

### CODE REVIEW

File: src/components/AddToCollectionModal/AddToCollectionModal.js [hunk 1]
```diff
@@ -35,6 +38,11 @@ const AddToCollectionModal = props => {
     }
   }, [newCollection]);
 
+  const notify = (collectionTitle, isChecked) => {
+    if (isChecked) toast(`Photo added to ${collectionTitle}`);
+    else toast(`Photo removed from ${collectionTitle}`);
+  };
```
Comment (yoloOnTheBattlefield) [inline src/components/AddToCollectionModal/AddToCollectionModal.js:45]: this looks like someone from kindergarten made it. Find another way (the idea is good, just it bothers me)

File: src/components/AddToCollectionModal/AddToCollectionModal.js [hunk 2]
```diff
@@ -54,7 +62,9 @@ const AddToCollectionModal = props => {
                   onChange={e =>
                     props.addToCollection(e.target.checked, e.target.value)
                   }
+                  onClick={e => notify(item.title, e.target.checked)}
                 />
+                <StyledSpan></StyledSpan>
```
Comment (yoloOnTheBattlefield) [inline src/components/AddToCollectionModal/AddToCollectionModal.js:68]: `<StyledSpan />`

File: src/components/FeaturedCollection/FeaturedCollection.js [hunk 3]
```diff
@@ -23,7 +23,7 @@ const FeaturedCollection = props => {
     <Container>
       <HeaderDiv>
         <h4 onClick={props.handleClick}>{props.title}</h4>
-        <StyledBinIcon /* onClick={props.deleteCollection} */ />
+        <StyledBinIcon onClick={props.deleteCollection} />
```
Comment (yoloOnTheBattlefield) [inline src/components/FeaturedCollection/FeaturedCollection.js:26]: I would also make this show just on hover

File: src/components/SearchResultsCollections/SearchResultsCollections.js [hunk 4]
```diff
@@ -25,6 +25,7 @@ const SearchResultsCollections = props => {
   }, [searchTerm]);
 
   useEffect(() => {
+    console.log('calling api');
```
Comment (yoloOnTheBattlefield) [inline src/components/SearchResultsCollections/SearchResultsCollections.js]: 😭

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/components/FeaturedCollection/FeaturedCollection.js (hunk 3 above)
Comment (aOrro) [reply src/components/FeaturedCollection/FeaturedCollection.js:26]: thats a good idea, when hovering on the title right?

**Pull request merged**